typedargs/typeinfo.py

```diff
@@ -235,6 +237,26 @@ def _is_factory(cls, typeobj):
 
         return False
 
+    @staticmethod
+    def get_type_name(type_class: type) -> Union[str, None]:
```

Comment:
For consistency with the rest of CoreTools and typedargs, `classmethod` is preferred.  The actual functionality in this case is the same but we've been using `classmethod` generally for this kind of thing.

Reply:
done